packages/common/src/types/visualizations.ts

```diff
@@ -17,18 +17,19 @@ export type BarChartConfig = {
     };
     axes: {
         x: {
-            reference?: string;
+            reference: string;
             label?: string;
         };
         y: {
             reference: string;
-            position: 'left' | 'right';
+            position?: 'left' | 'right';
             label: string;
         }[];
     };
     series: {
         reference: string;
         yIndex: number;
+        name: string;
```

Comment:
I believe the `label` of the `y` should be used instead of adding a `name` here.

So, for a serie X, I have a `yIndex`.
I can attribute a `label` to `y`
That should be updated and the `axes.y[serie.yIndex]` will get the `label` for us. That way you don't need to add `name`

LMK if you want to chat about this

Reply:
Updated.

I feel like we might end up going back to separating series and axis config cause this makes it so you can set series labels separately from the axes, but this is fine. Echarts separates axis and series config un propose so you can do things like name the axes differently than the series or have multiple series on an axis.